scripts/run-tests.js

```diff
@@ -0,0 +1,12 @@
+const os = require('os')
+const shell = require('shelljs')
+
+shell.config.fatal = true // throw if a command fails
+
+const isLinux = os.type() === "Linux"
```

Comment:
Can we do `isWindows` instead? This script is also used during development, and many contributors use mac.

Reply:
The `isLinux` variable is only used for ignoring the `buidler-vyper` tests in windows and mac. In windows, docker is available but defaulting to windows containers and changing it to linux containers is not straightforward (at least I wasn't able to do it easily). In mac, inside github actions, docker is not available for, I think, licensing reasons.